packages/icons/src/svg/logo-apple-colored.svg

```diff
@@ -0,0 +1,10 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g clip-path="url(#clip0_18856_34441)">
```

Comment:
Unnecessary clip-path. 

Jirka has already dealt with it in the Jobs Design System. I think that applying auto-layout to the frame with an icon fixed this issue…

Reply:
I contacted Jirka to try to fix it.

Reply:
it is fixed now :)